packages/Webkul/Product/src/Repositories/ProductCustomerGroupPriceRepository.php

```diff
@@ -4,9 +4,18 @@
 
 use Illuminate\Support\Str;
 use Webkul\Core\Eloquent\Repository;
+use Illuminate\Database\QueryException;
 
 class ProductCustomerGroupPriceRepository extends Repository
 {
+    /**
+     * Duplicate customer group price error code constant.
+     * 
+     * This constant is used to identify duplicate customer group price errors
+     * across different validation points in the application.
+     */
+    const DUPLICATE_CUSTOMER_GROUP_PRICE = 'duplicate_customer_group_price';
+
     /**
```

Comment:
Check this,
https://github.com/bagisto/bagisto/pull/10977#discussion_r2412851045